src/menu_events.cpp

```diff
@@ -1426,41 +1426,54 @@ void console_handler::do_droid()
 	// default to the current side if empty
 	const unsigned int side = side_s.empty() ? team_num_ : lexical_cast_default<unsigned int>(side_s);
 
+	utils::string_map symbols;
+	symbols["side"] = std::to_string(side);
+
 	if(side < 1 || side > menu_handler_.teams().size()) {
-		utils::string_map symbols;
-		symbols["side"] = side_s;
 		command_failed(VGETTEXT("Can't droid invalid side: '$side'.", symbols));
 		return;
 	} else if(menu_handler_.board().get_team(side).is_network()) {
-		utils::string_map symbols;
-		symbols["side"] = std::to_string(side);
 		command_failed(VGETTEXT("Can't droid networked side: '$side'.", symbols));
 		return;
-	} else if(menu_handler_.board().get_team(side).is_local_human()) {
-		utils::string_map symbols;
-		symbols["side"] = std::to_string(side);
-		if(menu_handler_.board().get_team(side).is_droid() ? action == "on" : action == "off") {
-			print(get_cmd(), VGETTEXT("Side '$side' is already droided.", symbols));
-			return;
+	} else if(menu_handler_.board().get_team(side).is_local()) {
+		bool changed = false;
+
+		if(action == "on" || action == "ON") {
```

Comment:
Instead of checking all upper and all lower I‘d rather allow any case combination. So you could for example just lowercase `action` and compare against the lowercase strings.

Reply:
Updated.